### ISSUE TITLE
Show blank instead of None in session/speaker forms if data is empty

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#2695]
Example:
![onpaste 20161222-213642](https://cloud.githubusercontent.com/assets/2404372/21431799/ca8105c4-c88e-11e6-96cd-cd905534bb49.png)

**Comment by Marauderer97** [comment]
@niranjan94 Can I try and fix this?

**Comment by Marauderer97** [comment]
![image](https://cloud.githubusercontent.com/assets/20185076/21451039/fa71d716-c921-11e6-952b-cfb18132e309.png)
When I try editing a speaker on my local host, I don't get any None value displayed @niranjan94

**Comment by niranjan94** [comment]
@Marauderer97 

Steps to reproduce:
1. Import the fossasia 2016 sample (https://github.com/fossasia/open-event/blob/master/sample/FOSSASIA16.zip)
2. Goto `/events/<event_id>/speakers/`
3. The issue would be seen

**Comment by magdalenesuo** [comment]
@niranjan94 hello.. While trying to upload, it gives an "internal server error" message
https://cloud.githubusercontent.com/assets/20799954/21452710/ce46dfb2-c92e-11e6-87b8-cad1ee7e844e.png

**Comment by niranjan94** [comment]
@magdalenesuo do you have celery and redis running on your local installation ? Both are required for imports to work.

**Comment by magdalenesuo** [comment]
@niranjan94 Thanks..got the error. Can I work on this?

**Comment by niranjan94** [comment]
@magdalenesuo sure .. you can

**Comment by magdalenesuo** [comment]
@niranjan94  While working on it, I found that if the field is empty , it stays empty after Save. Its just that it has been filled as 'None', and  if I remove the 'None' and save if, it stays blank.

https://cloud.githubusercontent.com/assets/20799954/21454656/303fedce-c940-11e6-9575-d5a9852441f0.png

**Comment by niranjan94** [comment]
@magdalenesuo then can you track at which point it get's saved as 'None' (string) ?

**Comment by magdalenesuo** [comment]
@niranjan94 I tried creating my own event and added a random speaker. There was no problem with None. I left some fields blank. It didn't get saved as 'None' anywhere.
I think the zip file we imported had already 'None' filled in all empty fields. otherwise, the portion of the code seems to be working fine as is :D 

(https://cloud.githubusercontent.com/assets/20799954/21456432/0e1401e6-c94e-11e6-9fa5-bb4eac9e31a8.png)

should I still try to track that point where None string gets saved?
